script/amqp_loader

```diff
@@ -1,44 +0,0 @@
-#!/usr/bin/env ruby
```

Comment:
And more stuff now handled by warren.
This *did* have the ability to hook a consumer directly onto a dead-letter queue, which was quite nice, but its easy enough to pop the message back onto the standard queue from the rabbitMQ admin panel. (Plus I've *never* used this feature)